Sources/Wendy/cli/commands/OSCommand.swift

```diff
@@ -428,15 +428,14 @@ struct OSCommand: AsyncParsableCommand {
 
             // Download and extract as separate progress bars when not using cache
             let imageDownloader = ImageDownloaderFactory.createImageDownloader()
-            let realDownloader = imageDownloader as! ImageDownloader
```

Comment:
weird... was this needed? was ImageDownloading protocol not enough to make the method calls?

Reply:
I don't know, I just wanted to get rid of this huge code smell